core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -451,7 +451,7 @@ public void addColumn(
             throwDistressException(err);
         }
 
-        txWriter.bumpStructureVersion(this.denseSymbolMapWriters);
+        bumpStructureVersion();
```

Comment:
Separate `bumpStructureVersion` with assert that _meta file structure version is updated before committing _txn structure version